sections/deckbuilding.tex

```diff
@@ -118,9 +118,9 @@ \subsection*{\pagetarget{spells}{Spell Cards}}
 Spell Cards\index{Spell Card} have three possible primary effects.
 Using the topmost, basic version of the Spell has no additional costs.
 To access the other effects, you may \textbf{Empower} a Spell\index{Empowering Spells} by paying the indicated cost (3) to get a more powerful outcome (4).
-You may pay this cost by playing other cards for their Empower \includesvg[height=10px]{\svgs/empower.svg} effect before casting the Spell.
-All Spell Cards also have an alternative bottom (5) \includesvg[height=10px]{\svgs/empower.svg} effect.\par
-
+You may pay this cost by playing other cards for their Empower \includesvg[height=10px]{\svgs/empower.svg} effect (e.g. the Spell Power Statistic) before casting the Spell.
+All Spell Cards also have an alternative bottom (5) \includesvg[height=10px]{\svgs/empower.svg} effect.
+You cannot preserve the played \includesvg[height=10px]{\svgs/empower.svg}: casting a Spell consumes all \includesvg[height=10px]{\svgs/empower.svg} played so far, and if you stop playing cards in sequence, the power dissipates.\par
```

Comment:
```suggestion
You cannot preserve the played \includesvg[height=10px]{\svgs/empower.svg}: casting a Spell consumes all the \includesvg[height=10px]{\svgs/empower.svg} played so far, and if you stop playing cards in sequence, the power dissipates.\par
```
Shouldn't it be like that?

Reply:
Likely. As Russian, I often miss English articles